shell/pom.xml

```diff
@@ -67,6 +67,11 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-compress</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.googlecode.java-diff-utils</groupId>
```

Comment:
from looking on github, there seems to be more recent releases of this library: https://github.com/java-diff-utils/java-diff-utils

you had mentioned about potentially being worried about CVEs regarding the use of a shared /tmp space; although our usage isn't going to be affected, it would still be flagged by any vulnerability scan, so we should try to use a version without CVEs if possible, otherwise we're producing more work down the road.